### ISSUE TITLE
Running MySQL on Mac

### ISSUE THREAD

**Issue by scripting** [scripting/Scripting-News#108]
Installing and running MySQL on Linux is quick and easy. But on a Mac? I can't seem to get it to start. I have it installed. 

Notes --

1. I want to access the database in Terminal. I don't want to use anything else.

2. I don't want MAMP. I just want MySQL.

3. I basically want what I have on Linux running on my Mac.

**Comment by dw808303** [comment]
Dave -- do you have either Homebrew or Macports on your Mac?

**Comment by scripting** [comment]
I've been really clear about what I want to do. Instead of answering suggestions that I do something else with "No I don't want to do that' -- I'm just going to delete them. Don't be offended.

**Comment by dw808303** [comment]
Not offended at all, just misunderstood the question.

**Comment by papascott** [comment]
I think it depends on how you installed it. If you installed the package from mysql.com, they put the binaries in /usr/local/mysql/bin. They also offer a [Preference Pane for MacOS ](https://dev.mysql.com/doc/refman/5.7/en/osx-installation-prefpane.html) for the MacOS System Preferences to start and stop the server. That might be the easiest way to set it up so the server always runs (just like in Linux 😃)

![image](https://user-images.githubusercontent.com/8681117/51804480-d3accc80-2261-11e9-9fb3-b173a9dc154b.png)

**Comment by scripting** [comment]
Hi Scott, yes I installed from mysql.com. And when I cd'd to that directory, that is where the mysql app is. I started it via the Control Panel, and then --

mysql --password

And when prompted entered the password I set up for root when I installed MySQL and got this error message.

ERROR 1045 (28000): Access denied for user 'davewiner'@'localhost' (using password: NO)

So I tried

sudo mysql --password

It asked for two passwords, the system password and the root user password for MySQL and voila I'm where I wanted to be.
